makefile

```diff
@@ -1,21 +1,23 @@
 .PHONY: init clear rebuild purge test lint lab ipython
 
+PYTHON=$(shell command -v python || command -v python3)
```

Comment:
I know MacOS doesn't use `bash` anymore, so @ajfriend would you mind running `command -v python` and letting me know what it prints out for you? I can't remember if that's a bash built-in or a binary usable by any shell.

Reply:
Works for me
```
bash-3.2$ command -v python
/Users/kbarron/.pyenv/shims/python
```

Reply:
I get the same as Kyle, and on `zsh`.